web/html/src/manager/content-management/list-filters/appstreams/appstreams.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+
+import React, {useState} from 'react';
+import {showErrorToastr} from 'components/toastr/toastr';
+import TextInput from './text-input';
+import SelectInput from './select-input';
+import Network from 'utils/network';
+
+export default function AppStreams() {
+  const [channels, setChannels] = useState([]);
+  const [isBrowse, setBrowse] = useState(false);
+  const [isLoading, setLoading] = useState(false);
+
+  const enableBrowse = () => {
+    setLoading(true);
+    Network.get("/rhn/manager/api/channels/modular", "application/json").promise
+      .then((channels) => {
+        setChannels(channels.data);
+        setLoading(false);
+        setBrowse(true);
+      }).catch(xhr => showErrorToastr(Network.responseErrorMessage(xhr)));
+  }
+
+  return (
+    isBrowse ?
+      <SelectInput channels={channels}/>
+    :
+      <React.Fragment>
```

Comment:
You can replace `<React.Fragment></React.Fragment>` by this short version `<></>`